plugins/GSdx/GSCrc.cpp

```diff
@@ -286,6 +287,7 @@ CRC::Game CRC::m_games[] =
 	{0x3E205556, SuikodenTactics, EU, 0},
 	{0xB808413B, SuikodenTactics, JP, 0},
 	{0xA33AF77A, TenchuFS, US, 0},
+	{0x89E63B6B, TenchuFS, RU, 0}, // Beta
```

Comment:
Can you move this under the EU version ? It's a small nitpick but I prefer not to have a PAL version in between NTSC versions. Maybe put it in between ES and EU.